### PR TITLE
Upgrade KCC operator to version 1.113.0

### PR DESCRIPTION
**Pull request by justinsb**
Generated using `make upgrade-kcc`

### CODE REVIEW

File: operator/channels/packages/configconnector/1.113.0/namespaced/per-namespace-components.yaml [hunk 1]
```diff
@@ -168,24 +164,6 @@ spec:
...
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        - name: POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        image: gke.gcr.io/prometheus-to-sd:v0.11.12-gke.0
-        name: prom-to-sd
```
Comment (jingyih) [inline operator/channels/packages/configconnector/1.113.0/namespaced/per-namespace-components.yaml:185]: Is removing "prom-to-sd" intentional?

Comment (justinsb) [reply operator/channels/packages/configconnector/1.113.0/namespaced/per-namespace-components.yaml]: Good catch - and @gemmahou caught this also!  We think it was caused by a newer version of kustomize; we should make sure that `make upgrade-kcc` (and everything) runs with a known version of kustomize

Comment (acpana) [reply operator/channels/packages/configconnector/1.113.0/namespaced/per-namespace-components.yaml]: ( automated PR also staging removal: [ref](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1142/files#diff-893e90058739be2dda3bb215a7605399edada7ef4a914c5d2013ad0335ee5d3fL185-L188)

**Comment by justinsb** [comment]
We hit the issue that should be fixed by https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1114  "timeout waiting for process kube-apiserver to stop" ; re-running tests

**Comment by justinsb** [comment]
Fixed up the problem with the kustomize version, removed the "WIP" designation.  Should be ready for review now!

**Review by jingyih — COMMENTED** [review]
/lgtm

**Pull request merged**